DSharpPlus/Clients/DiscordShardedClient.cs

```diff
@@ -540,6 +541,7 @@ private void HookEventHandlers(DiscordClient client)
             client.MessageReactionRemovedEmoji += this.Client_MessageReactionRemovedEmoji;
             client.WebhooksUpdated += this.Client_WebhooksUpdate;
             client.Heartbeated += this.Client_HeartBeated;
+            client.Zombied += this.Client_Zombied;
```

Comment:
Add this to UnhookEventHandlers too

Reply:
Will do this

Reply:
This has now been done in the latest commit